exercises/luhn/src/example.erl

```diff
@@ -19,6 +19,19 @@ checksum(Number) ->
 
 
 
+isNumeric(Number) -> 
+  case re:run(Number, "^[0-9 ]+$") of
+    {match, Captured} -> true;
+    nomatch    -> false
+  end.
+
+
+
+isMinLength(Number) -> 
+  re:replace(Number, "(^\\s+)|(\\s+$)", "", [global,{return,list}]).
```

Comment:
The name suggests a boolean return value, while `re:replace/4` returns a string. What is the purpose of this function and can you achieve the same goal with a fold or map over the input?

Also as said above, please use snake_case for function names.